frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SelectEmbedTypePane.tsx

```diff
@@ -112,27 +122,46 @@ export function SelectEmbedTypePane({
         </SharingPaneButton>
 
         {/* INTERACTIVE EMBEDDING */}
-        <Link
+        <MaybeLink
```

Comment:
When the embedding is disabled (interactive, SDK) instead of rendering the link we don't render it since it makes no sense to render `<a>` without `href` when the embedding is disabled.